SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -996,19 +1021,38 @@ void USpatialReceiver::HandleUnreliableRPC(Worker_ComponentUpdateOp& Op)
 				ObjectRef.Entity = Schema_GetEntityId(EventData, SpatialConstants::UNREAL_PACKED_RPC_PAYLOAD_ENTITY_ID);
 			}
 		}
-
+		
 		UObject* TargetObject = PackageMap->GetObjectFromUnrealObjectRef(ObjectRef).Get();
 
+		// TODO(Alex): Maybe queue by ObjectRef (create jira).
+		// It may happen when:
+		// 1. Packing RPCs (so that it's sent through a different entity)
+		// 2. Object has been destroyed (to fix that we need to clean queued component udpates for removed entity in void USpatialDispatcher::ProcessOps())
 		if (!TargetObject)
 		{
-			UE_LOG(LogSpatialReceiver, Warning, TEXT("HandleUnreliableRPC: Could not find target object: %s, skipping rpc at index: %d"), *ObjectRef.ToString(), Payload.Index);
+			UE_LOG(LogSpatialReceiver, Warning, TEXT("HandleRPC: Could not find target object: %s, skipping rpc at index: %d"), *ObjectRef.ToString(), Payload.Index);
```

Comment:
When does this happen?

Reply:
Added the comment above.

Reply:
Resolving since there's no pending action.